modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java

```diff
@@ -276,6 +281,12 @@ protected String getStatusMessage(int status, Locale locale) {
 			ddmFormInstanceVersionLocalService.getFormInstanceVersions(
 				ddmFormInstanceId, QueryUtil.ALL_POS, QueryUtil.ALL_POS, null);
 
+		ddmFormInstanceVersions = ListUtil.copy(ddmFormInstanceVersions);
```

Comment:
Why did you need to create this copy before sort the versions, @rodrigopaulino?

Reply:
Honestly, it's probably here because I picked this hunk of code from the 7.1.x branch where the sort of versions is correct. But I don't see why it's necessary either. I'll make sure to remove it from 7.1.x when backporting the fix.